### ISSUE TITLE
Koala on Rails wiki update

### ISSUE THREAD

**Issue by kevinjbayer** [arsduo/koala#587]
I was messing around updating a project to the 5.1 beta and received an error that alias_method_chain was deprecated. I tracked it down to usage in my koala.rb initializer. I think I've got a fix that I'd be happy to edit into the wiki, but just wanted some feedback if it is indeed correct. It doesn't break my app, but I haven't fully tested yet.

Original:
```
Koala::Facebook::OAuth.class_eval do
  def initialize_with_default_settings(*args)
    case args.size
      when 0, 1
        raise "application id and/or secret are not specified in the config" unless Facebook::APP_ID && Facebook::SECRET
        initialize_without_default_settings(Facebook::APP_ID.to_s, Facebook::SECRET.to_s, args.first)
      when 2, 3
        initialize_without_default_settings(*args) 
    end
  end 

  alias_method_chain :initialize, :default_settings 
end
```

My fix (per [justinweiss.com](http://www.justinweiss.com/articles/rails-5-module-number-prepend-and-the-end-of-alias-method-chain/):
```
module InitializeWithDefaultSettings
  def initialize(*args)
    case args.size
    when 0, 1
      raise 'application id and/or secret are not specified in the config' unless Facebook::APP_ID && Facebook::SECRET
      initialize_without_default_settings(Facebook::APP_ID.to_s, Facebook::SECRET.to_s, args.first)
    when 2, 3
      initialize_without_default_settings(*args)
    else
    end
  end
end

Koala::Facebook::OAuth.send(:prepend, InitializeWithDefaultSettings)
```

**Comment by kevinjbayer** [comment]
Posted too soon. My fix doesn't seem to work.

**Comment by arsduo** [comment]
Hmm. That wiki page was one I hadn't written myself and probably needs some real rewriting.

You can always accomplish the same thing with two alias calls or write a quick module that generated the OAuth objects using the appropriate values. There's a suggestion (#226) to add a global default, much like the Twitter gem, so I might implement that soon.

If you figure out a solution and want some feedback I'd be happy to give some! Otherwise we can close this, up to you.

**Comment by arsduo** [comment]
Good news! I just merged in #591, which gives a way to provide default values globally through `Koala.configure`. If you update to the new `3.0.0.beta3` release, you should be able to use it. (If you do upgrade, let me know how it works for you!)

**Comment by kevinjbayer** [comment]
Thanks, I'll give it a shot tomorrow!
